tests/ui/borrowck/suggestions/overloaded-index-not-mut-but-should-be-mut.stderr

```diff
@@ -0,0 +1,38 @@
+error[E0596]: cannot borrow `*string` as mutable, as it is behind a `&` reference
+  --> $DIR/overloaded-index-not-mut-but-should-be-mut.rs:7:5
+   |
+LL |     string.push_str("test");
+   |     ^^^^^^ `string` is a `&` reference, so the data it refers to cannot be borrowed as mutable
+   |
+help: consider using `get_mut`
+   |
+LL -     let string = &map[&0];
+LL +     let string = map.get_mut(&0).unwrap();
```

Comment:
It's a bit confusing that the suggestion is on a different line to the one mentioned earlier in the error. Is it possible to indicate that the line needing to be changed is on line 6?

Reply:
> Is it possible to indicate that the line needing to be changed is on line 6?

Yes, that is the LL annotation on the left, which gets normalized here but would have printed normally if it isn't in an UI test

Reply:
Oh right, that's fine then.